### PR TITLE
DOC: some rewording of highlights

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
@jreback I proofread the highlights and made some edits (but language is always a bit subjective). Comments?

### CODE REVIEW

File: doc/source/release.rst [hunk 1]
```diff
@@ -48,13 +48,20 @@ users upgrade to this version.
...
-- ``pd.test()`` top-level nose test runner is available (:issue:`4327`)
-- Adding support for a ``RangeIndex`` as a specialized form of the ``Int64Index`` for memory savings, see :ref:`here <whatsnew_0180.enhancements.rangeindex>`.
-- API breaking ``.resample`` changes to make it more ``.groupby`` like, see :ref:`here <whatsnew_0180.breaking.resample>`.
-- Removal of support for deprecated float indexers; these will now raise a ``TypeError``, see :ref:`here <whatsnew_0180.enhancements.float_indexers>`.
-- The ``.to_xarray()`` function has been added for compatibility with the `xarray package <http://xarray.pydata.org/en/stable/>`__, see :ref:`here <whatsnew_0180.enhancements.xarray>`.
-- Addition of the :ref:`.str.extractall() method <whatsnew_0180.enhancements.extract>`, and API changes to the :ref:`.str.extract() method <whatsnew_0180.enhancements.extract>` and :ref:`.str.cat() method <whatsnew_0180.enhancements.strcat>`.
+- Moving and expanding window functions are now methods on Series and DataFrame,
+  similar to ``.groupby``, see :ref:`here <whatsnew_0180.enhancements.moments>`.
```
Comment (jreback) [inline doc/source/release.rst]: double backticks on Series/DataFrame

**Comment by jreback** [comment]
trivial change. lgtm. merge away (I have a small doc fix w.r.t. the float indexers section but orthogonal)

**Comment by jorisvandenbossche** [comment]
I fixed the float indexers code block issue already on master, a colon was missing

**Comment by jreback** [comment]
oh ok, then i'll rebase on top

File: doc/source/release.rst (hunk 1 above)
Comment (jorisvandenbossche) [inline doc/source/release.rst]: We actually don't do that consistently, and write it more without than with backticks (at least in this file)

Comment (jreback) [inline doc/source/release.rst]: ok either way.

**Pull request closed without merging**